test/integration/reachability.go

```diff
@@ -43,19 +43,22 @@ func (r *reachability) run() error {
 		r.accessLogs[app] = make([]string, 0)
 	}
 
-	err := r.makeRequests()
-	if err != nil {
+	if err := r.makeRequests(); err != nil {
 		return err
 	}
 	if verbose {
 		log.Println("requests:", r.accessLogs)
 	}
 
-	if err = r.checkAccessLogs(); err != nil {
+	if err := r.checkAccessLogs(); err != nil {
 		return err
 	}
 
-	if err = r.checkMixerLogs(); err != nil {
+	if err := r.checkMixerLogs(); err != nil {
+		return err
+	}
+
+	if err := r.verifyTCPRouting(); err != nil {
```

Comment:
Sure, but as far as I could tell envoy did not generate HTTP access logs for HTTP traffic that was only handled at L4 (i.e. tcp_proxy).